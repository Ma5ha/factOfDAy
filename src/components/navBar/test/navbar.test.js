import React from "react";

import renderer from "react-test-renderer";

import NavBar from "../navbar";
import ToggleButton from "../../toggleButton/toggle";

it("shoud renderd darkBackground", () => {
  global.localStorage.setItem("theme", "dark");
  const nav = renderer
    .create(
      <NavBar>
        <ToggleButton />
      </NavBar>
    )
    .toJSON();
  expect(nav).toMatchSnapshot();
  console.log(nav);
});
