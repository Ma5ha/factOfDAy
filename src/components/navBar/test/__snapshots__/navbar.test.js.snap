// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sasas 1`] = `
<nav
  className="bottomMarginNav flexRow flexCenter whiteBackGround"
/>
`;

exports[`shoud renderd darkBackground 1`] = `
<nav
  className="bottomMarginNav flexRow flexCenter darkBackGround"
/>
`;
